### DISCUSSION TITLE
Some questions about velero.

### DISCUSSION THREAD

**Discussion by makabaka-boom** [vmware-tanzu/velero#3295] [Community support Q&A]
1. Now velero invokes the function "refresh" every 5 minutes to get api resources in K8S. Why not to get api resources at the time when backup started ?
2. In fact velero writes every single resource to temp files first during the process of backup. Then velero reads them from temp files  and handles them in memory. What is this way for ？Less memory usage? 
Would like to talk it with me？ Thanks a lot if you give some replies.

**Comment by codegold79** [comment]
@Lighting-wings, I don't have any answers, but maybe you can upvote an issue I wrote here: https://github.com/vmware-tanzu/velero/issues/3471. It is related to your first comment regarding Velero sometimes needing 5 minutes to recognize a new API resource.
